openlibrary/core/lending.py

```diff
@@ -335,15 +335,18 @@ def update_availability_schema_to_v2(v1_resp, ocaid):
     try:
         response = requests.get(url, timeout=config_http_request_timeout)
         items = response.json().get('responses', {})
-        for ocaid in items:
-            items[ocaid] = update_availability_schema_to_v2(items[ocaid], ocaid)
+        for pkey in items:
+            ocaid = items[pkey].get('identifier', key == 'identifier' and pkey)
+            items[pkey] = update_availability_schema_to_v2(items[pkey], ocaid)
         return items
     except Exception as e:  # TODO: Narrow exception scope
         logger.exception("get_availability(%s)" % url)
         items = { 'error': 'request_timeout', 'details': str(e) }
 
-        for ocaid in ids:
-            items[ocaid] = update_availability_schema_to_v2(
+        for pkey in ids:
+            # key could be isbn, ocaid, or openlibrary_[work|edition]                                          
```

Comment:
```suggestion
            # key could be isbn, ocaid, or openlibrary_[work|edition]
```
Trailing whitespace.